src/classes/storage.ts

```diff
@@ -1,36 +1,82 @@
-import { StopGroup, StopCluster, StopGroupGpuid, StopClusterGpuid } from '../types';
+import {
+  GroundPlacesObject,
+  StopGroup,
+  StopCluster,
+  StopGroupGpuid,
+  StopClusterGpuid,
+  GroundPlaceType,
+  Gpuid,
+  StopGroupProperties,
+  StopClusterProperties,
+} from '../types';
 
 /**
  * @description Manipulate GroundPlaces.
  */
 export class Storage {
+  private readonly groundPlaces: GroundPlacesObject;
```

Comment:
Object in a type name is not a good way to type something. Please rename it.

Reply:
Renamed to `GroundPlacesStored`, is it better?

Reply:
Don't like the naming either. Couldn't it be a `GroundPlace[]` ?
A `GroundPlace` being either a `StopGroup` or a `StopCluster`

Reply:
I was thinking about it but the problem is that I already have a class with this name, and it causes problems when I want to type through this interface when I already import or use the class of the same name!

Reply:
This is now renamed to GroundPlaces, and the main class is now renamed with `GroundPlacesManager` for now